.github/workflows/snyk.yml

```diff
@@ -0,0 +1,28 @@
+name: SNYK Analysis
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  scan:
+    runs-on: ubuntu-latest
+    steps:
+      # Checks-out under $GITHUB_WORKSPACE
+      - uses: actions/checkout@v4
+
+      # synk can do this, but it wants requirements.txt in the repo root
+      - name: Install requirements
+        run: pip install -r requirements.txt
+
+      - name: Snyk scan
+        uses: snyk/actions/python-3.10@master
+        env:
+          SNYK_TOKEN: 802c6bde-ee4b-4ca5-bbc0-9b4823a1fbae

```

Comment:
Hmmm shouldn't this be using ${{ secrets.SNYK_TOKEN }} instead?

Reply:
Yes it will.  This is for a non-RH junk/test account.  I have revoked the token now so no longer valid and will stick to secrets. :)